src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -352,16 +355,7 @@ namespace winrt::TerminalApp::implementation
         if (_startupState == StartupState::NotInitialized)
         {
             _startupState = StartupState::InStartup;
-            if (_startupActions.Size() == 0)
-            {
-                _OpenNewTab(nullptr);
-
-                _CompleteInitialization();
-            }
-            else
-            {
-                ProcessStartupActions(_startupActions, true);
-            }
```

Comment:
@zadjii-msft, @DHowett, @carlos-zamora please double check me on this.

I'm 99% sure that `AppCommandlineArgs::ValidateStartupCommands()` is already figuring out that there are no actions on startup and makes the default tab open up. And then this AGAIN does the same thing and is likely vestigial. 

My problem is that for starting the COM server, I want there to be no actions other than just getting the frame up. When I had it crafted a few revisions ago as an action, everything was fine because it was following the bottom route here. But then y'all had me take the action out as it was admittedly too much. However, it then sent me down the top path here which is making a default tab just for funsies.

Basically, I believe it's safe to pull this and send everything into the `ProcessStartupActions` handler where it will then process 0 actions (since I yoinked the early return) and finish with `_CompleteInitialization()` always.

Confirm/deny for me?

Reply:
From looking into it a bit, it looks like the following scenario wouldn't work:
- opening WT without any `startupActions` and no handoff
This would _not_ open a new tab on startup, when it should? So wouldn't it just open and close itself immediately?

If that scenario works, I say you're good.

Reply:
ValidateStartupCommands makes a new tab action if there are no actions when the arg parse is done. So I don't think it's a problem.